cognite/client/_api/transformations/schema.py

```diff
@@ -15,13 +15,21 @@ class TransformationSchemaAPI(APIClient):
     _RESOURCE_PATH = "/transformations/schema"
 
     def retrieve(
-        self, destination: TransformationDestination, conflict_mode: Optional[str] = None
+        self,
+        destination: TransformationDestination,
+        conflict_mode: Optional[str] = None,
+        with_instance_space: Optional[bool] = None,
+        is_connection_definition: Optional[bool] = None,
+        instance_type: Optional[str] = None,
     ) -> TransformationSchemaColumnList:
         """`Get expected schema for a transformation destination. <https://docs.cognite.com/api/v1/#operation/getTransformationSchema>`_
 
         Args:
             destination (TransformationDestination): destination for which the schema is requested.
             conflict_mode (Optional[str]): conflict mode for which the schema is requested.
+            with_instance_space (Optional[bool]): Is instance space set at the transformation config or not
+            is_connection_definition (Optional[bool]): If the edge is a connection definition or not
```

Comment:
What is this parameter? It does not make sense to me, even with the explanation.

Reply:
withInstanceSpace - # optional, if specified it takes  precedence over the space mentionled in the sql query

 isConnectionDefinition -  if you pass the query `isConnectionDefinition=true`, we won’t send back the type as part of the schema